src/shared/components/oncoprint/ResultsViewOncoprint.tsx

```diff
@@ -507,8 +507,8 @@ export default class ResultsViewOncoprint extends React.Component<
             },
             get ngchmButtonActive() {
                 return AppConfig.serverConfig.show_mdacc_heatmap &&
-                    (self.props.store.remoteNgchmUrl.result &&
```

Comment:
[Don't require parens for same-operator logical expressions](https://prettier.io/blog/2019/11/09/1.19.0.html#dont-require-parens-for-same-operator-logical-expressions-6864-by-jridgewell)